server/models/Transaction.js

```diff
@@ -27,7 +24,42 @@ export default (Sequelize, DataTypes) => {
       }
     },
 
-    // stores the currency that the transaction happened in (currency of the host)
+    UserId: {
+      type: DataTypes.INTEGER,
+      references: {
+        model: 'Users',
+        key: 'id'
+      },
+      onDelete: 'SET NULL',
+      onUpdate: 'CASCADE',
+      allowNull: false
+    },
+
+    // Keeps a reference to the host because this is where the bank account is
+    // Note that the host can also change over time (that's why just keeping GroupId is not enough)
+    HostId: {
+      type: DataTypes.INTEGER,
+      references: {
+        model: 'Users',
+        key: 'id'
+      },
+      onDelete: 'SET NULL',
+      onUpdate: 'CASCADE',
+      allowNull: false
+    },
+
+    GroupId: {
+      type: DataTypes.INTEGER,
+      references: {
+        model: 'Groups',
+        key: 'id'
+      },
+      onDelete: 'SET NULL',
+      onUpdate: 'CASCADE',
+      allowNull: false
```

Comment:
I ran the migration script locally and it worked fine.

Shouldn't a transaction always have a GroupId and HostId?
Maybe we can make GroupId optional to account for host fees in the future as a separate row?
The question is whether we want to make a host a collective in an on itself (which I think we should).